Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM madnificent/ember:5.3.0 as builder
+FROM node:20.12 AS builder
 
 LABEL maintainer="info@redpencil.io"
 
-ARG SHOW_APP_VERSION_HASH=false
```

Comment:
Removed this ARG since it's not used by the app, and it is a remnant from the OP codebase.